### PR TITLE
DON'T MERGE YET: Seaice DA, needs the GDASApp PR merged first. 

### PR DESCRIPTION
**Pull request by guillaumevernieres**
- [ ] We need to add a seaice analysis restarts to the IC's

### CODE REVIEW

File: ush/forecast_postdet.sh [hunk 1]
```diff
@@ -524,9 +524,15 @@ CICE_postdet() {
     local seconds
     seconds=$(to_seconds "${restart_date:8:2}0000")  # convert HHMMSS to seconds
     cice_restart_file="${DATArestart}/CICE_RESTART/cice_model.res.${restart_date:0:4}-${restart_date:4:2}-${restart_date:6:2}-${seconds}.nc"
+    if [[ "${DO_JEDIOCNVAR:-NO}" = "YES" && "${restart_date}" = "${sdate_current_cycle}" ]]; then
```
Comment (JessicaMeixner-NOAA) [inline ush/forecast_postdet.sh]: @guillaumevernieres Two things, one i changed the sdate_current_cycle name on you to be: model_start_date_current_cycle  -- sorry! 

And 2, I don't think you can have RERUN=YES if restart_date=model_start_date_current_cycle, so I don't think this is a  needed block.... but we should get someone else to confirm.

Comment (guillaumevernieres) [reply ush/forecast_postdet.sh]: arf ... yes, you also told me about it. I'll remove the block too.

Comment (guillaumevernieres) [reply ush/forecast_postdet.sh]: did you push that change @JessicaMeixner-NOAA ? I don't see `model_start_date_current_cycle` defined anywhere.

Comment (JessicaMeixner-NOAA) [reply ush/forecast_postdet.sh]: just pushed.. sorry i thought i had

**Comment by JessicaMeixner-NOAA** [comment]
Closing... This branch is now in develop.

**Pull request closed without merging**